sensorlib/ReadMe.md

```diff
@@ -0,0 +1,88 @@
+# CARLA Sensor Library
+
+## Overview
+
+The sensor library provides a proxy sensor for the CARLA simulator, exposing additional features.
+
+This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a
+CARLA-driven simulation. The data is intended to feed into a CARMA V2X system for data fusion with other sensors and
+action by CARMA platform vehicles.
+
+The SimulatedSensor acts as a wrapper to the CARLA Semantic Lidar Sensor, capturing the data and performing
+post-processing to generate a list of detected objects upon request. The primary reason for processing data from CARLA
+is to ensure noise is applied to the output signal, and customization is possible for occlusion and any other procedures
+in the future. The native CARLA sensor does not provide signal noise, misdetection simulation, or customization.
+
+Raw sensor observations from the CARLA sensor are analyzed to build a dataset representing the current objects in scene.
+
+## Carla CDA Sim Adapter
+
+The Carla CDA Sim Adapter exposes the sensing capability through an RPC service. See
+the [Carla CDA Sim Adapter Documentation](doc/CarlaCDASimAdapter.md) for details.
+
+## Carla CDA Sim API
```

Comment:
This is not what I meant. I meant rename the XML RPC service to CARLA CDA Sim Adapter. The library should still be called  what ever we are calling the sensor library. I assume this is the API you are referring to here.